docs/advanced/worker-commandline-reference.md

```diff
@@ -1,14 +1,14 @@
 ---
-title: "Worker Cmdline Reference"
+title: "Worker cmdline reference"
 layout: default
 sort: 3
 ---
 
-# NFD-Worker Commandline Flags
+# NFD-Worker commandline flags
```

Comment:
```suggestion
# Nfd-Worker commandline flags
```

Should we treat acronyms differently? Meaning if we use NFD should we have it all capitalized also in a header or do you want to apply the  `Nfd` style?

Reply:
Oh, these ones. Yes, definitely not *NFD-Worker*, I'd say. Either *NFD-worker* or *Nfd-worker*. In this particular case here we're talking about the name of the binary which is `nfd-worker` so in that sense *Nfd-worker* could make sense. If NFD would be used as a separate abbreviation I'd keep it fullcaps. WDYT @zvonkok? I'm open to suggestions.

Reply:
Maybe stupid but what about, 

# The `nfd-worker` commandline flags

Reply:
### The `nfd-worker` cofinguration file reference

Reply:
> The `nfd-worker` commandline flags

Hmm, I'm not a native English-speaker, but, there's something bothering me there 🤔 

How about 
- Command line flags of nfd-master
- Command line flags of nfd-worker
- Configuration reference of nfd-worker
??

Reply:
Yep lets do that, makes more sense, as I said maybe but its definitely `stupid`.  :zany_face:

Reply:
Changed to `Commandline flags of nfd-worker`